integration_test/gce/gce_testing.go

```diff
@@ -1731,3 +1733,33 @@ func RunForEachPlatform(t *testing.T, f func(t *testing.T, platform string)) {
 func ArbitraryPlatform() string {
 	return strings.Split(os.Getenv("PLATFORMS"), ",")[0]
 }
+
+func AddTagToVm(ctx context.Context, logger *log.Logger, vm *VM, tag string) (CommandOutput, error) {
```

Comment:
I think we should return an error if the input tag contains a comma, because that will confuse gcloud.